test/selenium/sessions_ui_test.js

```diff
@@ -124,11 +112,8 @@ describe('sessions ui', function() {
         });
 
       it('should accept matching, long-enough passwords', function() {
-          return setUpPage.typePassword('12345678').then(function() {
-              return setUpPage.confirmTypePassword('12345678');
-          }).then(function(){
-              return setUpPage.tapSubmitButtonSignUp();
-          }).then(function(successfulPageView) {
+          return setUpPage.successLogin(12345678, 12345678)
+          .then(function(successfulPageView) {
               return successfulPageView.successLogin();
```

Comment:
Nit: `successLogin` sounds ambiguous now that we have `setUpPage.successLogin`. I'd change it to something closer to `loginMessage`